### PR TITLE
Enhancement/97/get awsmeta more than once

### PR DESCRIPTION
**Pull request by rclagett**
Addresses #97

### CODE REVIEW

File: src/diamond/handler/netuitive_handler.py [hunk 1]
```diff
@@ -277,32 +277,36 @@ def _add_aws_meta(self):
             resp = urllib2.urlopen(request, timeout=1).read()
             j = json.loads(resp)
 
-            for k, v in j.items():
-                if type(v) is list:
-                    vl = ', '.join(v)
-                    v = vl
-                self.element.add_attribute(k, v)
+            if j and self.aws_meta_state < 1:
```
Comment (TheConnMan) [inline src/diamond/handler/netuitive_handler.py]: Why not use a boolean?

File: src/diamond/handler/netuitive_handler.py [hunk 2]
```diff
@@ -277,32 +277,36 @@ def _add_aws_meta(self):
             resp = urllib2.urlopen(request, timeout=1).read()
             j = json.loads(resp)
```
Comment (TheConnMan) [inline src/diamond/handler/netuitive_handler.py]: This outside the if statement means you always load the endpoint, but don't always do anything with it. Probably don't want to fetch it if you don't have to.

Comment (rclagett) [reply src/diamond/handler/netuitive_handler.py]: Made a change to only request the endpoint when it's necessary.

File: src/diamond/handler/netuitive_handler.py (hunk 1 above)
Comment (rclagett) [reply src/diamond/handler/netuitive_handler.py]: Is there a benefit to using a boolean over the current implementation?

Comment (TheConnMan) [reply src/diamond/handler/netuitive_handler.py]: The variable's purpose is easier to understand. I searched for a few seconds to see other references for this variable to see if other values had other interpretations.

**Review by TheConnMan — APPROVED** [review]

**Pull request merged**